mcs/class/corlib/System.Threading/Thread.cs

```diff
@@ -52,8 +52,6 @@ sealed class InternalThread : CriticalFinalizerObject {
 		IntPtr handle;
 		IntPtr native_handle; // used only on Win32
 
-		/* Note this is an opaque object (an array), not a CultureInfo */
-		private object cached_culture_info;
 		/* accessed only from unmanaged code */
```

Comment:
It would be better to change this to __unused_something, and not bump the corlib version. Corlib version bumps are kinda painful since they require a new monolite, makes it harder to match runtime and corlibs from different sources, etc.